feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-single-selection-base.component.ts

```diff
@@ -0,0 +1,144 @@
+import { ConfiguratorAttributeBaseComponent } from './configurator-attribute-base.component';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Configurator } from '../../../../core';
+import { ConfigFormUpdateEvent } from '../../../form';
+import {
+  ConfiguratorAttributeQuantityComponentOptions,
+  ConfiguratorAttributeQuantityService,
+  Quantity,
+} from '../../quantity';
+import { map } from 'rxjs/operators';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  template: '',
```

Comment:
Do we need that, or can we omit template completely?

Reply:
I will check it

Reply:
Without @Component it does not work, get the following expection: